packages/devtools_app/lib/src/memory/memory_controller.dart

```diff
@@ -52,7 +52,13 @@ class MemoryController extends DisposableController
 
   static const logFilenamePrefix = 'memory_log_';
 
-  bool showHeatMap = true;
+  final _showHeatMap = ValueNotifier<bool>(false);
+
+  ValueListenable<bool> get showHeatMap => _showHeatMap;
+
+  void toggleShowHeatMap(bool value) {
```

Comment:
why not just expose showHeatMap as a ValueNotifier if we are fine with users reading and writing the value?